rebar.config

```diff
@@ -43,3 +44,5 @@
 
 {post_hooks, [{"(linux|darwin)", clean, "make -C apps/aecore/c_src clean"}
              ]}.
+
+{dialyzer, [{warnings, [unknown]}]}.
```

Comment:
I suggest not to include this until a fix for `0: Unknown type http_uri:uri/0` is proposed.

Reply:
I fixed that

Reply:
Thanks, looks good.